content/php-api-using-jwt/index.md

```diff
@@ -0,0 +1,226 @@
+---
+title: "Securing PHP API using JWT and Apache"
+date: "2021-08-4"
+coverImage: "jwt.jpg"
+author: "Harikrishna Kundariya"
+tags: ["Securing PHP API", "PHP API using JWT","PHP API using Apache"]
+Description: "Harikrishna Kundariya, a marketer, developer, IoT, ChatBot & Blockchain savvy, designer, co-founder, Director of [eSparkBiz Technologies](www.esparkinfo.com) @SaaS App Development Company. His 8+ experience enables him to provide digital solutions to new start-ups based on IoT and ChatBot. "
+---
+Security is one of the most important things that you need to look at while developing an application. As people become more aware and hackers become more notorious, you need to employ systems that strengthen your application's data.
+
+Previously, people used session storage to secure their applications. In recent times, sessions have proved inefficient, and that made people migrate to authentication with APIs. Even though this was a superb and robust way to secure web applications, it became obsolete as hackers tried to figure out how to crack this authentication.
+
+As the web evolves to accept more and more users, the research for secure authentication techniques speeds up. It was in 2010 when the world was introduced to a new and secure authentication standard, JWT. Let's know more about JWT.
+
+**What is JWT?**
```

Comment:
Please use heading markdown formatting for all the headings and sub-headings instead of bold text

Reply:
Done